client/flutter/lib/components/page_scaffold/page_header.dart

```diff
@@ -24,6 +25,7 @@ class PageHeader extends StatelessWidget {
     this.appBarColor,
     this.inSliver = true,
     this.trailing,
+    this.primary = true,
```

Comment:
Helpful to add comments on the purpose of `heroTag`, `inSliver`, `trailing` and `primary` as that wasn't obvious to me upon reading the code.